src/transactions/management/transactionWorker.ts

```diff
@@ -157,12 +165,21 @@ export class TransactionWorker {
             // transaction sent to chain
             this.sentTransactions.push([sentTransaction.value.hash, sequenceNumber, null]);
             // check sent transaction execution
+            this.emit(
+              TransactionWorkerEvents.TransactionSent,
+              `transaction hash ${sentTransaction.value.hash} has been commited to chain`,
```

Comment:
Is this the data being emitted? If so, shouldn't we emit something structured instead? That way the callback can actually do something with the data. So it emits some kind of `TransactionSentEventData` that has a `txnHash` field. The user can use that in the callback too rather than `data: any`.

Reply:
Good call! I have updated the worker/management to have a types returned data